vision/text/detect_words.py

```diff
@@ -58,6 +61,9 @@ def detect_russian_word(
         detected_words = self.tessdata["text"]
         box_obs = []
 
+        rows, columns = color_image.shape[0], color_image.shape[1]
```

Comment:
Instead of indexing, you can do this:
```suggestion
        rows, columns, _ = color_image.shape
```